docs/src/userguide.md

```diff
@@ -336,6 +346,12 @@ Note that some of the arithmetic operations have been extended for
 [`HomogeneousPolynomial`](@ref); division, for instance, is not extended.
 The same convention used for `Taylor1` objects is used when combining
 `TaylorN` polynomials of different order.
+Both `HomogeneousPolynomial` and `TaylorN` are equiped with a total *lexicographical*
+order, provided by overloading [`isless`](@ref).
+The ordering is consistent with the interpretation that there are infinitessimal
+elements in the algebra; for details see M. Berz, "Automatic Differentiation as
+Nonarchimedean Analysis", Computer Arithmetic and Enclosure Methods, (1992), Elsevier,
+439-450.
```

Comment:
I like the example provided for `Taylor1`. Can the same be done here?

Reply:
I just pushed a commit addressing this point. Let me know what you think